builder/hcloud/helper.go

```diff
@@ -0,0 +1,21 @@
+package hcloud
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/packer-plugin-sdk/multistep"
+	packersdk "github.com/hashicorp/packer-plugin-sdk/packer"
+)
+
+// errorHandler is a helper function to reduce the amount of bloat and complexity
+// caused by redundant error handling logic.
+func errorHandler(state multistep.StateBag, ui packersdk.Ui, prefix string, err error) multistep.StepAction {
+	wrappedError := err
```

Comment:
Why do we need to introduce a new variable here?

Reply:
I find it more explicit.